backend/Squirrel.ConsoleApp.BL/Interfaces/IClientIdFileService.cs

```diff
@@ -0,0 +1,8 @@
+﻿
```

Comment:
redundant space here and in `ClientIdFileService.cs` and `ConnectionFileService.cs` and in `IExecuteOnClientSide.cs` and in `QueryParameters.cs`

and could you please add notes how to call methods in `HubController.cs` (url example etc)?